### DISCUSSION TITLE
Slingbox M1 Custom Remote lost after reboot.

### DISCUSSION THREAD

**Discussion by highveld66** [GerryDazoo/Slinger#593] [General]
I havw just got an M1. Factory reset it. Set it up with desktop software. All was good. Added a custom remote. This worked well.
But when I power off and turn back on the M1. The slingbox works fine for video source. But the remote does not work any more.
It seems to have lost or wiped the custom remote..
Any ideas?

Thanks

**Comment by CatsBy119** [comment]
That was my experience too. Went through all the steps (posted elsewhere) and waited for the Desktop software to go back to the home screen. Works great until the Slingbox M1 power is cycled. I repeated the steps a number of times over several hours but could not get the custom remote to "stick".

**Reply by highveld66** [reply]
I tried all sorts of steps too to try and make remote settings stay. But every time when reboots the remote is lost.
Is this being done by a bug or.....

**Comment by highveld66** [comment]
Got a new M1 fro ebay. Setup and got exactly same behaviour. No remote after reboot after setup
I think it is linked to a firware update when setting up?

**Comment by pfuller88** [comment]
I'm not alone.  This has happened to my M1 several times.  I have re-set it up,  several time, it is finicky.

**Reply by tatsuo777github** [reply]
Has anyone figured this out ? I purchased brand new M1s and M2s but cannot get them setup completely. After setting remote, the desktop software does not finish setup and once the unit is turned off, the remote setting as well as name of the box is wiped out.

**Comment by tatsuo777github** [comment]
I tested five new M1s and m2s and all of them do not retain the remote setting after setup. The desktop Slinbox app does not seem to finish the initialization. It is not surprising as the server is shutdown but for anyone who tries to use a new slingbox hardware problem. We rely on the desktop software to setup the hardware and it can be very problematic going forward.
